src/main/java/uk/ac/cam/cl/dtg/isaac/api/GameboardsFacade.java

```diff
@@ -169,15 +158,15 @@ public final Response generateTemporaryGameboard(@Context final HttpServletReque
                                                    @QueryParam("levels") final String levels,
                                                    @QueryParam("concepts") final String concepts,
                                                    @QueryParam("questionCategories") final String questionCategories) {
-    List<String> subjectsList = splitCsvStringQueryParam(subjects);
-    List<String> fieldsList = splitCsvStringQueryParam(fields);
-    List<String> topicsList = splitCsvStringQueryParam(topics);
+    List<String> subjectsList = questionManager.splitCsvStringQueryParam(subjects);
```

Comment:
You should be accessing these via the class reference, not the instance reference (so the `Q` in`questionManager` should be uppercase).